python/graphstorm/gconstruct/file_io.py

```diff
@@ -27,6 +27,40 @@
 import torch as th
 import h5py
 
+def read_index_json(data_file):
+    """ Read the index from a JSON file.
+
+    Each row is a JSON object that contains an index.
+
+    Parameters
+    ----------
+    data_file : str
+        The JSON file that contains the index.
+
+    Returns
+    -------
+    Numpy array : the index array.
+    """
+    with open(data_file, 'r', encoding="utf8") as json_file:
+        indices = []
+        for line in json_file.readlines():
+            indices.append(json.loads(line))
```

Comment:
Can you provide an example in either README or https://github.com/awslabs/graphstorm/blob/main/docs/source/tutorials/own-data.rst?

Reply:
modifying this tutorial requires much more effort. let's table this.